src/transactions/feed/queryHelper.ts

```diff
@@ -157,10 +142,7 @@ export function useFetchTransactions(): QueryHookResult {
           }
           // If there are new transactions update transactions in redux and fetch balances
           if (hasNewTransaction) {
-            batch(() => {
```

Comment:
not directly related but we no longer need to manually dispatch the token balances fetch, this is done in the home saga already